internal/services/oracle/oracle_exa_db_vm_cluster_resource.go

```diff
@@ -0,0 +1,634 @@
+						ForceNew: true,
+					},
+				},
+			},
+		},
+
+		"domain": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		},
+
+		"license_model": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(exadbvmclusters.LicenseModelBringYourOwnLicense),
+				string(exadbvmclusters.LicenseModelLicenseIncluded),
+			}, false),
+		},
+
+		"nsg_cidrs": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"destination_port_range": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"max": {
+									Type:     pluginsdk.TypeInt,
+									Optional: true,
+									Computed: true,
+									ForceNew: true,
+								},
+
+								"min": {
+									Type:     pluginsdk.TypeInt,
+									Optional: true,
+									Computed: true,
+									ForceNew: true,
+								},
+							},
+						},
+					},
+					"source": {
+						Type:     pluginsdk.TypeString,
+						Optional: true,
+						Computed: true,
+						ForceNew: true,
+					},
+				},
+			},
+		},
+
+		"private_zone_ocid": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		},
+
+		"scan_listener_port_tcp": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			Default:      1521,
+			ForceNew:     true,
+			ValidateFunc: validation.IntBetween(1024, 8999),
+		},
+
+		"scan_listener_port_tcp_ssl": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			Default:      2484,
+			ForceNew:     true,
+			ValidateFunc: validation.IntBetween(1024, 8999),
+		},
+
+		"system_version": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ValidateFunc: validation.StringMatch(regexp.MustCompile(`^(19|22|23|24|25)\.[0-9]+(\.[0-9]+)*|[0-9]+(\.[0-9]+)$`),
+				"The system_version must match following patterns: (19|22|23|24|25).x(.x(.x)).",
+			),
+		},
+
+		"time_zone": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		},
+
+		"tags": commonschema.Tags(),
+
+		"zones": commonschema.ZonesMultipleRequiredForceNew(),
+	}
+}
+
+func (ExadbVmClusterResource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"hostname_actual": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"ocid": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+	}
+}
+
+func (ExadbVmClusterResource) ModelObject() interface{} {
+	return &ExadbVmClusterResource{}
+}
+
+func (ExadbVmClusterResource) ResourceType() string {
+	return "azurerm_oracle_exa_db_vm_cluster"
+}
+
+func (r ExadbVmClusterResource) Create() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 24 * time.Hour,
```

Comment:
Is this specified on purpose, since per your test result, it doesn't need so long time?